.npmignore

```diff
@@ -1,7 +0,0 @@
-src
-test
-examples
-.DS_Store
-*.log
-*.tgs
-*.yml
```

Comment:
I added `files` in package.json which pretty much instead of having to ignore what we don't want, we explicitly add what we do want. Therefore no need to have `.npmignore` anymore.